File: Lista de Exercícios 1/Calculadora Simples.java
/*
Faça um algoritmo para ler dois inteiros (variáveis A e B) e efetuar as operações de adição, subtração, multiplicação e divisão de A por B apresentando ao final os quatro resultados obtidos.

Entrada
A entrada contém 2 valores inteiros.

Saída
Imprimir a operação e o resultado da operação para cada operação a ser executada. Cuide para que tenha um espaço antes e depois do sinal de igualdade, conforme o exemplo abaixo e  não esqueça de imprimir o fim de linha após o resultado. Para casos em que o resultado seja real, imprima com 2 casas decimais.
*/

import java.util.*;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n1= sc.nextInt();
        int n2= sc.nextInt();
        System.out.printf("SOMA = %d\n", n1+n2);
        System.out.printf("SUBTRACAO = %d\n", n1-n2);
        System.out.printf("MULTIPLICACAO = %d\n", n1*n2);
        if (n2==0){
            System.out.printf("DIVISAO = INDETERMINADO\n");
        }
        else{
            float result=(float)n1/(float)n2;
            System.out.printf("DIVISAO %.2f\n", result);
        }
    }
}

File: Lista de Exercícios 1/Calculadora Simples.c
/*
Faça um algoritmo para ler dois inteiros (variáveis A e B) e efetuar as operações de adição, subtração, multiplicação e divisão de A por B apresentando ao final os quatro resultados obtidos.

Entrada
A entrada contém 2 valores inteiros.

Saída
Imprimir a operação e o resultado da operação para cada operação a ser executada. Cuide para que tenha um espaço antes e depois do sinal de igualdade, conforme o exemplo abaixo e  não esqueça de imprimir o fim de linha após o resultado. Para casos em que o resultado seja real, imprima com 2 casas decimais.
*/

#include <stdio.h>

int main(void) {
    int n1=0;
    int n2=0;
    scanf("%d",&n1);
    scanf("%d",&n2);
    printf("SOMA = %d\n", n1+n2);
    printf("SUBTRACAO = %d\n", n1-n2);
    printf("MULTIPLICACAO = %d\n", n1*n2);
    if (n2==0){
        printf("DIVISAO = INDETERMINADO\n");
    }
    else{
        float result=(float)n1/(float)n2;
        printf("DIVISAO %.2f\n", result);
    }
    return 0;
}
